.github/workflows/downstream-ci-hpc.yml

```diff
@@ -1345,6 +1358,63 @@ jobs:
           ${{ needs.setup.outputs.findlibs || (needs.setup.outputs.use_master == 'True' && 'findlibs:ecmwf/findlibs@master') || 'findlibs:ecmwf/findlibs@develop' }}
           ${{ needs.setup.outputs.earthkit-utils || (needs.setup.outputs.use_master == 'True' && 'earthkit-utils:ecmwf/earthkit-utils@main') || 'earthkit-utils:ecmwf/earthkit-utils@develop' }}
         python_toml_opt_dep_sections: tests
+  anemoi-cascade:
```

Comment:
Don't we want to keep there commented out and only keep earthkit-workflows? Just to avoid impacting the other packages.

Reply:
Ah OK never mind this is not the test triggers here.